### DISCUSSION TITLE
How to do custom secret scan for binary file

### DISCUSSION THREAD

**Discussion by bultau** [aquasecurity/trivy#5046] [Q&A] [answered]
### Question

I'd like to do custom secret scan for binary file.
Here is my sample configuration.
This could not match any binary file but match only text file.

```
$ cat ./secret-config/trivy-secret.yaml
rules:
  - id: Test for Keys in this Project
    category: Passwords
    title: API Key Issue
    severity: MEDIUM
    regex: ^.
```

And here is my command.
```
trivy fs --scanners secret --secret-config ./secret-config/trivy-secret.yaml /data
```

Thanks for your help.

### Target

None

### Scanner

Secret

### Output Format

None

### Mode

Standalone

### Operating System

wsl2

### Version

```bash
$ trivy --version
Version: 0.44.1
```

**Comment by knqyf263** [comment, marked as answer]
Unfortunately, Trivy doesn't support binaries in secret scanning.
